internal/cli/finalizer_test.go

```diff
@@ -0,0 +1,205 @@
+package cli
```

Comment:
Please add `t.Parallel()` as the first line of tests whenever possible.

Reply:
👍